plugins/meta/flannel/flannel.go

```diff
@@ -182,7 +206,7 @@ func cmdAdd(args *skel.CmdArgs) error {
 
 	if !hasKey(n.Delegate, "ipMasq") {
 		// if flannel is not doing ipmasq, we should
-		ipmasq := !fenv.ipmasq
+		ipmasq := !*fenv.ipmasq
```

Comment:
did I miss this change in the previous review? if not, did it actually compile!?